recipes/zed/Cargo.toml

```diff
@@ -0,0 +1,682 @@
+# This file is generated by `cargo hakari`.
+# To regenerate, run:
+#     cargo install cargo-hakari
```

Comment:
If this is auto-generated by these commands, then please run these commands during the build instead of checking in this file.

Reply:
Ah, this was a workaround for a rattler-build bug. Let me remove it real quick